src/main/java/com/sainttx/banknotes/command/BanknotesCommand.java

```diff
@@ -61,13 +62,27 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
                     sender.sendMessage(plugin.colorMessage(plugin.getConfig().getString("messages.invalid-number")));
                 } else {
                     ItemStack banknote = plugin.createBanknote(sender.getName(), amount);
-                    target.getInventory().addItem(banknote);
-                    target.sendMessage(plugin.colorMessage(plugin.getConfig().getString("messages.note-received")
-                            .replace("[money]", plugin.formatDouble(amount))
-                            .replace("[player]", sender.getName())));
-                    sender.sendMessage(plugin.colorMessage(plugin.getConfig().getString("messages.note-given")
-                            .replace("[money]", plugin.formatDouble(amount))
-                            .replace("[player]", target.getName())));
+                    
```

Comment:
No problem. Saw the reviews on your spigot resource and the most recent three reviews all requested the decimal change, and I wanted this on our network so I figured I'd give it a shot. Kinda sad how long it took for me to piece this together though xD.


Also personal reminder when I look at this in the morning to support below $1,000 and above $999,999.